rtil/src/native/engine.rs

```diff
@@ -0,0 +1,10 @@
+use std::cell::Cell;
+use crate::native::UeScope;
+use crate::native::uworld::ENGINE_INDEX;
+
+pub fn set_gamma(gamma: f32) {
+    UeScope::with(|scope| {
+        let object = scope.get(ENGINE_INDEX.get().unwrap());
+        object.get_field("DisplayGamma").unwrap::<&Cell<f32>>().set(gamma);
+    })
+}
```

Comment:
Create an `enum UEngine {}` for which you implement its static associated functions, similar to how it's done for `FSlateApplication`.